[2021-05-09 18:01:12,350] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-08T05:44:22.392437+00:00 [queued]>
[2021-05-09 18:01:12,381] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-08T05:44:22.392437+00:00 [queued]>
[2021-05-09 18:01:12,381] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:01:12,381] {taskinstance.py:1069} INFO - Starting attempt 1 of 3
[2021-05-09 18:01:12,381] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:01:12,521] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): python_push_into_target> on 2021-05-08T05:44:22.392437+00:00
[2021-05-09 18:01:12,525] {standard_task_runner.py:52} INFO - Started process 6457 to run task
[2021-05-09 18:01:12,530] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'python_push_into_target', '2021-05-08T05:44:22.392437+00:00', '--job-id', '125', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmp78m2ua6z', '--error-file', '/tmp/tmpqgn7vld1']
[2021-05-09 18:01:12,531] {standard_task_runner.py:77} INFO - Job 125: Subtask python_push_into_target
[2021-05-09 18:01:12,670] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.python_push_into_target 2021-05-08T05:44:22.392437+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-09 18:01:12,804] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=python_push_into_target
AIRFLOW_CTX_EXECUTION_DATE=2021-05-08T05:44:22.392437+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-08T05:44:22.392437+00:00
[2021-05-09 18:01:13,627] {logging_mixin.py:104} INFO - push_test
[2021-05-09 18:01:13,627] {python.py:118} INFO - Done. Returned value was: None
[2021-05-09 18:01:13,634] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=python_push_into_target, execution_date=20210508T054422, start_date=20210509T123112, end_date=20210509T123113
[2021-05-09 18:01:13,808] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-09 18:01:13,825] {local_task_job.py:146} INFO - Task exited with return code 0
